xmipy/xmiwrapper.py

```diff
@@ -160,6 +160,15 @@ def get_component_name(self) -> str:
         self.execute_function(self.lib.get_component_name, byref(component_name))
         return component_name.value.decode("ascii")
 
+    def get_version(self) -> str:
+        try:
+            len_version = self.get_constant_int("BMI_LENVERSION")
+            version = create_string_buffer(len_version)
+            self.execute_function(self.lib.get_version, byref(version))
```

Comment:
Does this throw an exception if the symbol is not there?
If yes, I think it's fine to let the caller deal with that.

Reply:
thx, will change that